delta/testkit/src/main/scala/ch/epfl/bluebrain/nexus/testkit/IOValues.scala

```diff
@@ -28,9 +28,17 @@ trait IOValues extends Assertions {
   )(implicit s: Scheduler = Scheduler.global): Future[Assertion] =
     future.map(_ => succeed)
 
+  implicit final def uioValuesSyntax[A](uio: UIO[A]): UIOValuesOps[A] = new UIOValuesOps(uio)
```

Comment:
why doesn't this yield a conflict? (because UIO[A] == IO[Nothing, A])

Reply:
No classtag for E (Nothing for UIO) ?

Which is why I added this part because it worked for other tests but for the ones in the tests of ProjectsDummy
```uio.accepted``` wouldn't compile